### PR TITLE
Fix some openmod warning including Harmony that show in plugins

### PR DESCRIPTION
**Pull request by rube200**

### CODE REVIEW

File: framework/OpenMod.Core/Plugins/OpenModPluginBase.cs [hunk 1]
```diff
@@ -34,7 +38,10 @@ public abstract class OpenModPluginBase : IOpenModPlugin, IAsyncDisposable
         public IRuntime Runtime { get; }
         public IEventBus EventBus { get; }
         protected ILogger Logger { get; set; }
+
+#if NETFRAMEWORK && !NET20
```
Comment (Trojaner) [inline framework/OpenMod.Core/Plugins/OpenModPluginBase.cs:42]: This is not acceptable, Harmony is supported on all platforms not just .NET Framework

File: framework/OpenMod.Core/OpenMod.Core.csproj [hunk 2]
```diff
@@ -18,24 +18,27 @@
     <EmbeddedResource Include="openmod.yaml" />
   </ItemGroup>
 
-  <ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'net35' or ($(TargetFramework.StartsWith('net4')))">
```
Comment (Trojaner) [inline framework/OpenMod.Core/OpenMod.Core.csproj:21]: not acceptable

**Review by Trojaner — CHANGES_REQUESTED** [review]

File: framework/OpenMod.Core/Plugins/OpenModPluginBase.cs (hunk 1 above)
Comment (rube200) [reply framework/OpenMod.Core/Plugins/OpenModPluginBase.cs:42]: i didn't added the other platforms because atm om dont compile to them

**Comment by Trojaner** [comment]
Please remove all platform conditions

**Pull request merged**